packages/@orbit/jsonapi/src/jsonapi-source.ts

```diff
@@ -27,6 +27,12 @@ if (typeof Orbit.globals.fetch !== 'undefined' && Orbit.fetch === undefined) {
   Orbit.fetch = Orbit.globals.fetch;
 }
 
+declare module '@orbit/core/dist/types/main' {
```

Comment:
I'm wary of this approach (including the module's path, including `dist`, feels wrong) but I need to better understand it and the alternatives.